### PR TITLE
std: Better formatting of tuple types

### PR DESCRIPTION
**Pull request by LemonBoy**
Skip the useless type name and the numeric field names.

### CODE REVIEW

File: lib/std/fmt.zig [hunk 1]
```diff
@@ -2164,6 +2180,9 @@ test "struct" {
     };
 
     try expectFmt("S{ .a = 456, .b = error.Unused }", "{}", .{inst});
+    // Tuples
+    try expectFmt("{ }", "{}", .{.{}});
```
Comment (MasterQ32) [inline lib/std/fmt.zig:2184]: I would add another test case for single-item tuples as those are another edge case (no trailing comma)

Comment (LemonBoy) [reply lib/std/fmt.zig:2184]: Good idea, added one (and force-pushed).
